reconcile/test/test_vpc_peerings_validator.py

```diff
@@ -58,3 +61,11 @@ def test_validate_no_internal_to_public_peerings_valid_internal(
     assert query_data.clusters is not None
     query_data.clusters[0].peering.connections[0].cluster.internal = True  # type: ignore[index,union-attr]
     assert validate_no_internal_to_public_peerings(query_data) is True
+
+
+def test_validate_no_public_to_public_peerings_valid(
```

Comment:
Worth testing where this returns `True`?

Reply:
added proper tests